### PR TITLE
README: document make alembic-new-migration

### PR DESCRIPTION
**Pull request by jgehrcke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,11 @@ May modify files. Analogue to what CI requires.
 It requires for some commands to be available in your current shell.
 Dependencies can be installed with `pip install -r requirements-dev.txt`.
 
+* `make alembic-new-migration`: Attempts to generate a new migration Python module in `migrations/versions/`.
+Requires setting the environment variable `ALEMBIC_MIGRATION_NAME` before invocation.
+Example: `export ALEMBIC_MIGRATION_NAME='repo_url_lenth'`.
+After the file was created you may want to change its permissions re-format it with `black`.
+
```
Comment (jgehrcke) [inline README.md:117]: btw, I already use the one-sentence-per-line idea, for better documentation diffs.

**Comment by jgehrcke** [comment]
ci/testsuite failed with known instability: https://github.com/conbench/conbench/issues/895

File: README.md [hunk 2]
```diff
@@ -110,6 +110,11 @@ May modify files. Analogue to what CI requires.
 It requires for some commands to be available in your current shell.
 Dependencies can be installed with `pip install -r requirements-dev.txt`.
 
+* `make alembic-new-migration`: Attempts to generate a new migration Python module in `migrations/versions/`.
+Requires setting the environment variable `ALEMBIC_MIGRATION_NAME` before invocation.
+Example: `export ALEMBIC_MIGRATION_NAME='repo_url_lenth'`.
+After the file was created you may want to change its permissions re-format it with `black`.
```
Comment (austin3dickey) [inline README.md]:
```suggestion
After the file was created you may want to change its permissions and re-format it with `black`.
```

**Review by austin3dickey — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jgehrcke) [reply README.md]: Thanks!

**Pull request merged**